src/docfx/docset/Docset.cs

```diff
@@ -59,6 +59,11 @@ internal class Docset
         /// </summary>
         public HashSet<Document> BuildScope => _buildScope.Value;
 
+        /// <summary>
+        /// Gets the config file name.
+        /// </summary>
+        public string ConfigFile { get; }
```

Comment:
>ConfigFile [](start = 22, length = 10)

ConfigFileName?

Reply:
Other places uses `XXXPath`, so `ConfigPath`?

Reply:
ConfigPath like a relative path, but it;s OK

---
In reply to: [225480012](https://github.com/dotnet/docfx/pull/3498#discussion_r225480012) [](ancestors = 225480012)

Reply:
ConfigPath is something else, would take ConfigFilaName here as below:
```csharp
public static (List<Error> errors, Config config, string configFileName) Load(string docsetPath, CommandLineOptions options, bool extend = true, RestoreMap restoreMap = null)
```